ebi_eva_common_pyutils/variation/contig_utils.py

```diff
@@ -0,0 +1,113 @@
+# Copyright 2020 EMBL - European Bioinformatics Institute
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import re
+import requests
+from retry import retry
+
+
+# TODO: Might be a good idea to re-visit this after a production implementation
+#  of the contig-alias resolution project is available
+@retry(tries=10, delay=5, backoff=1.2, jitter=(1, 3))
+def resolve_contig_accession_to_chromosome_name(contig_accession, line_limit=100):
```

Comment:
Are we sure this function will provide information that is not already in the assembly report.
This code looks very complicated for simply finding an alias and I'm not sure we really need it.

Reply:
Yes. This is the python-version of the logic that is currently being used in the website and has been reviewed previously. In fact, we explicitly reach for this in EVA-2150 detection script only when [we are out of luck with finding the chromosome name in the assembly report](https://github.com/EBIvariation/eva-tasks/pull/12/files#diff-8fee2a738e1f8916fe0967dc0af81c14R93).